RIGImageGallery.podspec

```diff
@@ -13,7 +13,7 @@ Pod::Spec.new do |s|
   s.license          = { :type => 'MIT', :file => 'LICENSE' }
   s.author           = { "Michael Skiba" => "mike.skiba@raizlabs.com" }
   s.source           = { :git => "https://github.com/raizlabs/RIGImageGallery.git", :tag => s.version.to_s }
-  s.social_media_url = 'https://twitter.com/ateliercw'
+  s.social_media_url = 'https://twitter.com/raizlabs'
```

Comment:
#? Wouldn't we want this to be someone who is likely to respond, i.e. you or another RZ OSS maintainer? Not sure if whoever is monitoring the company account will be able to field queries about this. It should probably match what's in the readme, so if you don't want to be on call for support, let's pick someone who is.